test/TestServerProjectsLowLevel/header/Generated/HeaderClient.cs

```diff
@@ -52,7 +52,6 @@ public HeaderClient(AzureKeyCredential credential, Uri endpoint = null, HeaderCl
         /// <summary> Send a post request with header value &quot;User-Agent&quot;: &quot;overwrite&quot;. </summary>
         /// <param name="userAgent"> Send a post request with header value &quot;User-Agent&quot;: &quot;overwrite&quot;. </param>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="userAgent"/> is null. </exception>
```

Comment:
Is `userAgent` something we'd want to special case?  It seems like more of a client parameter than a server parameter.

Reply:
Why?

Reply:
We can discuss this separately - let's not block this PR on it.